tests/test_cadquery.py

```diff
@@ -24,7 +25,8 @@
 )
 
 # test data directory
-testdataDir = os.path.join(os.path.dirname(__file__), "testdata")
+testdataDir = Path(os.path.dirname(__file__), "testdata")
```

Comment:
Using only pathlib it could be:
```py
testdataDir = Path(__file__).parent.joinpath("testdata")
testFont = str(testdataDir.joinpath("OpenSans-Regular.ttf"))
```

Reply:
IMHO the last version `Path('a')/Path('b')` is optimal.

Reply:
It looks fine.  My comment was more about that if you want to move to the pathlib form you can eliminate os.path.dirname.